x-pack/plugin/snapshot-repo-test-kit/qa/azure/build.gradle

```diff
@@ -45,3 +46,20 @@ tasks.named("javaRestTest") {
 tasks.register("azureThirdPartyTest") {
   dependsOn "javaRestTest"
 }
+
+tasks.register("managedIdentityJavaRestTest", RestIntegTestTask) {
+  testClassesDirs = sourceSets.javaRestTest.output.classesDirs
+  classpath = sourceSets.javaRestTest.runtimeClasspath
+}
+
+tasks.named("managedIdentityJavaRestTest") {
+  systemProperty 'test.azure.fixture', Boolean.toString(useFixture)
```

Comment:
FWIW, all of this can go up above in the `register` block and you can get rid of this extra configuration block.

Reply:
Hmm that was the first thing I tried, but it didn't work. I'll try again in a follow-up.

Reply:
The behavior _should_ be identical.

Reply:
Hm yeah seems ok now, not sure what I did wrong last time: https://github.com/elastic/elasticsearch/pull/111444